spec/features/admin_matching_attributes_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.feature "Admin checking a claims" do
+  let(:user_id) { "userid-345" }
```

Comment:
I think that if we have no intention of varying the user ID or verifying that it is used anywhere, then we can get rid of this let and use the default value provided by `sign_in_to_admin_with_role`.